src/main.js

```diff
@@ -94,6 +111,22 @@ function viewSaved() {
   randomCoverButton.classList.add("hidden");
   saveCoverButton.classList.add("hidden");
   homeButton.classList.remove("hidden");
+  for (var i = 0; i < savedCovers.length; i++) {
```

Comment:
In refactoring, we should tease this function apart so that goHome is only taking us back to the home page